endoreg_db/views/api/patient_finding_views.py

```
@@ -0,0 +1,317 @@
+            
+            # Prüfe ob Choice zur Classification gehört
+            if not location_classification.choices.filter(id=location_choice_id).exists():
+                return Response(
+                    {'error': 'location_choice gehört nicht zur location_classification'},
+                    status=status.HTTP_400_BAD_REQUEST
+                )
+            
+            # Erstelle Location
+            location = finding.add_location(location_classification_id, location_choice_id)
+            
+            serializer = PatientFindingLocationSerializer(location)
```

Ruff (0.11.9)

Undefined name `PatientFindingLocationSerializer`

(F821)